something-nice/nice-story

```diff
@@ -0,0 +1 @@
+Good lucky! Tomorrow
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.